src/wallet3/rpc/commands.h

```diff
@@ -2481,33 +2481,31 @@ namespace wallet::rpc {
-
 When specifying owners, either a wallet (sub)address or standard ed25519 public key is supported per mapping. Updating the value that a name maps to requires one of the owners to sign the update transaction. For wallets, this is signed using the (sub)address's spend key.
 
 For more information on updating and signing see the ONS_UPDATE_MAPPING documentation.)";
 
     struct REQUEST
     {
-      std::string        type;            // The mapping type: "session", "lokinet", "lokinet_2y", "lokinet_5y", "lokinet_10y", "wallet".
-      std::string        owner;           // (Optional): The ed25519 public key or wallet address that has authority to update the mapping.
-      std::string        backup_owner;    // (Optional): The secondary, backup public key that has authority to update the mapping.
-      std::string        name;            // The name to purchase via Oxen Name Service
-      std::string        value;           // The value that the name maps to via Oxen Name Service, (i.e. For Session: [display name->session public key],  for wallets: [name->wallet address], for Lokinet: [name->domain name]).
+      std::string        type;               // The mapping type: "session", "wallet", "lokinet", "lokinet_2y", "lokinet_5y", "lokinet_10y".
+      std::string        owner;              // (Optional): The ed25519 public key or wallet address that has authority to update the mapping.
+      std::string        backup_owner;       // (Optional): The secondary, backup public key that has authority to update the mapping.
+      std::string        name;               // The name to purchase via Oxen Name Service
+      std::string        value;              // The value that the name maps to via Oxen Name Service, (i.e. For Session: [display name->session public key],  for wallets: [name->wallet address], for Lokinet: [name->domain name]).
 
-      uint32_t           account_index;   // (Optional) Transfer from this account index. (Defaults to 0)
-      std::set<uint32_t> subaddr_indices; // (Optional) Transfer from this set of subaddresses. (Defaults to 0)
-      uint32_t           priority;        // Set a priority for the transaction. Accepted values are: or 0-4 for: default, unimportant, normal, elevated, priority.
-      bool               get_tx_key;      // (Optional) Return the transaction key after sending.
-      bool               do_not_relay;    // (Optional) If true, the newly created transaction will not be relayed to the oxen network. (Defaults to false)
-      bool               get_tx_hex;      // Return the transaction as hex string after sending (Defaults to false)
-      bool               get_tx_metadata; // Return the metadata needed to relay the transaction. (Defaults to false)
+      uint32_t           account_index;      // (Optional) Transfer from this account index. (Defaults to 0)
+      std::vector<uint32_t> subaddr_indices; // (Optional) Transfer from this set of subaddresses. (Defaults to 0)
```

Comment:
A `set` here seems to better agree with the purpose; is there a reason to change this to vector?

Reply:
omq doesnt like sets. Think this has happened with all the std::sets that were previously in our rpc layer

Reply:
It probably wouldn't be hard to add that to `param_parser.hpp`, but if this is the only use then probably not worth the bother.